.pre-commit-config.yaml

```diff
@@ -7,7 +7,7 @@ repos:
       - id: helm-docs-built
         args:
           # Comma separated list, no space
-          - --chart-to-generate=gpt-survey-summarizer
+          - --chart-to-generate=gpt-survey-summarizer,submission-report
```

Comment:
**suggestion:** Good thinking adding the helm-chart for README auto-generation.

I looks like the pre-commit hooks are not configured on your machine though.
From the git repo directory, you would want to run:
```
pre-commit install
```

This would wire the hook between `git` and `pre-commit` to run the hooks when creating a commit.

To force to run the hooks manually on all files, you can run:
```
pre-commit run  --all-files
```

This should generate the `README.md` file for your chart, that you can then add to your PR.

Reply:
:facepalm: ofc it runs locally... Thx, fixed it and committed